### DISCUSSION TITLE
Cannot download Texas Chainsaw Massacre Audio Files: "SDL2 not found...".

### DISCUSSION THREAD

**Discussion by Ashyslashy2000** [gildor2/UEViewer#310] [Q&A]
Hi all, I have been using UEViewer to access the Texas Chainsaw Massacre Game files and everything has been fine so far apart from the audio files. I am wanting to export the Johnny voicelines from the game, but unfortunately I am getting a lot of errors. I have attached screenshots below so you can see the errors I have been getting. Any help on this would be hugely appreciated! Thanks in advance. 
![image](https://github.com/gildor2/UEViewer/assets/135978159/87969ae6-7db7-436f-8c48-52e070d11bce)
![image](https://github.com/gildor2/UEViewer/assets/135978159/2868a3b5-9574-4389-8728-714f0bf5460f)
![image](https://github.com/gildor2/UEViewer/assets/135978159/bdf93664-5f31-4bb2-b1e8-6bd63b7bc58c)
![image](https://github.com/gildor2/UEViewer/assets/135978159/f68a9c09-74de-41ea-8dbc-d9ab02db7259)

**Comment by Blenux** [comment]
You need the SDL2.dll file to be able to use uModel properly.

**Reply by Ashyslashy2000** [reply]
Hi Blenux,

Many thanks for your reply. 

I have the SDL2.dll file installed as I manually checked the lib folder, and installed a brand new SDL2 file just to be sure so I'm not sure why it would still me causing this issue? 

Cheers, 

Ash.
